src/custom/hooks/useWalletInfo.ts

```diff
@@ -65,7 +65,7 @@ function checkIsSupportedWallet(params: {
 async function getWcPeerMetadata(connector: WalletConnectConnector): Promise<{ walletName?: string; icon?: string }> {
   const provider = (await connector.getProvider()) as WalletConnectProvider
 
-  const meta = provider.walletMeta
+  const meta = provider.signer.connection.wc.peerMeta
```

Comment:
i think you need to create an `||`  check here as the conditional on line `69` is failing and causing wallets to not appear.

`const meta = provider.walletMeta || provider.signer.connection.wc.peerMeta`

Reply:
Done